spec/spec_helper.rb

```diff
@@ -19,3 +19,13 @@ def app(app = nil, &blk)
   @app ||= block_given? ? app.instance_eval(&blk) : app
   @app ||= Padrino.application
 end
+# Added by @rodcul based on:
+# https://github.com/makersacademy/chitter-challenge/blob/master/spec/spec_helper.rb
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.